veggieseasons/pubspec.yaml

```diff
@@ -11,9 +11,26 @@ dependencies:
     sdk: flutter
 
   cupertino_icons: ^0.1.2
+  intl: ^0.15.7
+  scoped_model: ^0.3.0
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
 
 flutter:
+  assets:
+  - assets/images/placeholder.png
+  fonts:
+  - family: NotoSans
+    fonts:
+    - asset: assets/fonts/NotoSans-Regular.ttf
```

Comment:
Are you really using all these font variants? Just checking.

Reply:
Fair point. I have used them all at one point or another, but might not be right now. Let me get the rest of the app through review, and I'll double check at that point.